### DISCUSSION TITLE
NOAA Data for Pakistan cc: YASIR

### DISCUSSION THREAD

**Discussion by nquarder** [cu-esiil-edu/2024-data-short-course#76] [General]
@chyasirabbasjutt - here is a link to the NOAA Climate Data Online tool with results for Pakistan.

https://www.ncei.noaa.gov/cdo-web/datasets/GHCND/locations/FIPS:PK/detail#stationlist

Others who are interested in seeing how to search the NOAA/NCEI tool for other countries or locations can watch this video:
https://cuboulder.zoom.us/rec/share/9Z0gBWa2uQau58X_uqI9MRW6ofE7TawB9-kv-nKVopM_yePTqC6XrmNADawJ8YzA.Y04sTKj-obiSzOY9?startTime=1714316266000

We will add this to the learning portal soon, but for now you can see a bit more about how we're using the NOAA/NCEI API to request data for locations other than Boulder, CO. Hope this helps!

**Comment by chyasirabbasjutt** [comment]
Thank sir sharing the information.
